### PR TITLE
Create standardized PR templates for contracts

### PR DESCRIPTION
**Pull request by ith-harvey**
This PR creates three github PR templates that should be followed:
* `logic_src_changes.md`
* `cosmetic_src_changes.md`
* `testing_or_misc_nonsrc_changes.md`

To learn more about how GH templates work -> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

**Review by grandizzy — COMMENTED** [review]
I find this a good start and we can improve as we go if needed. Don't see though where changes for gas optimization should be included, like this PR #542

**Comment by ith-harvey** [comment]
> I find this a good start and we can improve as we go if needed. Don't see though where changes for gas optimization should be included, like this PR #542

Hmm think it was already included but cleaned up language -> https://github.com/ajna-finance/contracts/pull/539/commits/8fa357ab4751fc2518c3f5ec45d91cd40fa45ecb

**Review by grandizzy — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE/cosmetic_src_changes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* <FILE_NAME> (i.e.`src/ERC20Pool.sol`)
+  * <DESCRIP_OF_CHANGE> (i.e. changes in writes and reads)
+
+# Contract size
+## Pre Change
+<PASTE_OUTPUT_HERE>
+## Post Change
+<PASTE_OUTPUT_HERE>
+
+# Gas optimization:
```
Comment (EdNoepel) [inline .github/PULL_REQUEST_TEMPLATE/cosmetic_src_changes.md]: Recommend replacing `optimization` to `cost` in this and the other template.  Also to eliminate the `:` where inconsistent.

File: .github/PULL_REQUEST_TEMPLATE/testing_or_misc_nonsrc_changes.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Description of change
+## High level:
```
Comment (EdNoepel) [inline .github/PULL_REQUEST_TEMPLATE/testing_or_misc_nonsrc_changes.md]: nit: inconsistent `:`

**Review by EdNoepel — CHANGES_REQUESTED** [review]
Any PR which changes `src/` should include a field citing auditor approval for the change.  This will likely need to evolve once multiple auditors are in play.

**Comment by ith-harvey** [comment]
> Any PR which changes `src/` should include a field citing auditor approval for the change. This will likely need to evolve once multiple auditors are in play.

Based off of our conversation with Jack in the Sherlock channel I don't think this is necessary **YET** but should be added after TOB's start date 13th of March

File: .github/PULL_REQUEST_TEMPLATE/cosmetic_src_changes.md (hunk 1 above)
Comment (ith-harvey) [reply .github/PULL_REQUEST_TEMPLATE/cosmetic_src_changes.md]: fixed -> https://github.com/ajna-finance/contracts/pull/539/commits/824d45765aa782bf1507076fbf601f5ff252203b

File: .github/PULL_REQUEST_TEMPLATE/testing_or_misc_nonsrc_changes.md (hunk 2 above)
Comment (ith-harvey) [reply .github/PULL_REQUEST_TEMPLATE/testing_or_misc_nonsrc_changes.md]: fixed -> https://github.com/ajna-finance/contracts/pull/539/commits/824d45765aa782bf1507076fbf601f5ff252203b

**Review by EdNoepel — APPROVED** [review]

**Pull request merged**